modules/installation-alibaba-config-yaml.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-customizations.adoc
+
+:_content-type: REFERENCE
+[id="installation-alibaba-config-yaml_{context}"]
+= Sample customized `install-config.yaml` file for Alibaba Cloud
```

Comment:
No markdown in headings, so this should be:
 Sample customized install-config.yaml file for Alibaba Cloud

Reply:
Fixed here. Will fix n 8 other files.